common/zt_logSanitizer_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright © Microsoft <wastore@microsoft.com>
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package common
+
+import (
+	chk "gopkg.in/check.v1"
+)
+
+type logSanitizerSuite struct{}
+
+var _ = chk.Suite(&logSanitizerSuite{})
+
+func (s *logSanitizerSuite) TestLogSanitizer(c *chk.C) {
+
+	cases := []struct {
+		raw               string
+		expectedSanitized string
+	}{
+		{"string with no secrets", "string with no secrets"},
+
+		// DON'T redact these
```

Comment:
I'd recommend adding a test where `sig` is a value, actually.

I don't *think* it will fail, but, it's best to just make sure that it won't.

Reply:
Noted, hope to get to it tomorrow, if not earlier.